src/test/java/seedu/address/logic/parser/getparsers/GetFloorNumberCommandParserTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.parser.getparsers;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.getcommands.GetFloorNumberCommand;
+import seedu.address.model.person.FloorNumberPredicate;
+
+public class GetFloorNumberCommandParserTest {
+
+    private GetFloorNumberCommandParser parser = new GetFloorNumberCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        // empty input
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, GetFloorNumberCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_invalidFloorNumber_throwsParseException() {
```

Comment:
Additional checks for alphanumeric commands or commands with a lot of spaces in between would be helpful too!